src/Microsoft.TemplateEngine.Utils/EngineEnvironmentSettings.cs

```diff
@@ -13,10 +13,12 @@ namespace Microsoft.TemplateEngine.Utils
     public static class EngineEnvironmentSettings
     {
         private static readonly string DefaultLocale = "en_US";
+        private static readonly string DefaultAssemblyName = "None";
+        private static readonly Version DefaultAssemblyVersion = new Version("0.0.0.1");
 
         static EngineEnvironmentSettings()
         {
-            Host = new DefaultTemplateEngineHost(string.Empty, DefaultLocale);
+            Host = new DefaultTemplateEngineHost(string.Empty, DefaultLocale, DefaultAssemblyName, DefaultAssemblyVersion);
```

Comment:
Agree, this should go away. It was a cheap way to get around the problem of not always having the locale when the host first needs to be invoked. I added an DefaultTemplateEngineHost.UpdateLocale method to do it more properly.